x-pack/plugins/security_solution/cypress/README.md

```diff
@@ -115,8 +115,42 @@ cd x-pack/plugins/security_solution
 CYPRESS_BASE_URL=http(s)://<username>:<password>@<kbnUrl> CYPRESS_ELASTICSEARCH_URL=http(s)://<username>:<password>@<elsUrl> CYPRESS_ELASTICSEARCH_USERNAME=<username> CYPRESS_ELASTICSEARCH_PASSWORD=password yarn cypress:run:firefox
 ```
 
+#### CCS Custom Target + Headless
+
+This test execution requires two clusters configured for CCS. See [Search across clusters](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-cross-cluster-search.html) for instructions on how to prepare such setup.
+
+The instructions below assume:
+* Search cluster is on server1
+* Remote cluster is on server2
+* Remote cluster is accessible from the search cluster with name `remote`
+* Security and TLS are enabled
+
+```shell
+# bootstrap Kibana from the project root
+yarn kbn bootstrap
+
+# launch the Cypress test runner with overridden environment variables
+cd x-pack/plugins/security_solution
+CYPRESS_ELASTICSEARCH_USERNAME="user" \
+CYPRESS_ELASTICSEARCH_PASSWORD="pass" \
+CYPRESS_BASE_URL="https://user:pass@server1:5601" \
+CYPRESS_ELASTICSEARCH_URL="https://user:pass@server1:9200" \
+CYPRESS_CCS_KIBANA_URL="https://user:pass@server2:5601" \
+CYPRESS_CCS_ELASTICSEARCH_URL="https://user:pass@server2:9200" \
+CYPRESS_CCS_REMOTE_NAME="remote" \
+yarn cypress:run:ccs
```

Comment:
@banderror the configuration mentioned in that piece of the README is in case you want to use your own CCS setup, right now if you want to execute the automation from your local, you need to setup the CCS configuration and pass the variables. Doing something like `yarn cypress:run-as-ci` command that starts an setups an instance for you, it is out of scope for now for this PR.  But of course, these are just the first steps and we will continue working to make everything easier to setup :)

Reply:
@banderror once https://github.com/elastic/integration-test/pull/489 is merged you can use [integration-test/jenkins_test.sh](https://github.com/elastic/integration-test/blob/master/jenkins_test.sh) in this way:

```shell
VMS=ubuntu16_tar_ccs_cypress ./jenkins-test.sh
```

It sets up the two clusters with CCS and runs the Cypress CCS tests. The two clusters are in the same VM, managed with vagrant, and a snapshot is taken before the tests are executed. You can always restore the snapshot and run your tests again.

If your tests clean up after themselves, you don't need to restore the snapshot; just re-run `yarn cypress:run:ccs`.

Reply:
@MadameSheema gotcha, sounds good. Definitely would be nice to have something like `yarn cypress:run-as-ci`, but for CCS, for local development.